zeta/zeta.py

```diff
@@ -16,6 +16,11 @@
 ZETA_SRC_DIR = "."
 ZETA_INCLUDE_DIR = "."
 
+class YamlRefLoader(yaml.SafeLoader) :
+    def __init__(self, stream) :
+        super(YamlRefLoader, self).__init__(stream)
+    def ref(self, node):
+        return self.construct_scalar(node)
```

Comment:
I could not see any warning or error to the user when something is wrong. Is there anything related to it?

Reply:
I think that there isn't. Ref function will be called just when some "!ref" prefix appears in YAML file, and your responsibility is just remove "!ref" text to ZetaCLI deal with it later. If some warning or error must be put about it I think that the right location would be in construct_yaml function. We can discuss about that.

Reply:
Ok. Let's talk about this later. :+1: